new_client/src/components/about-us/AboutUs.jsx

```diff
@@ -14,10 +14,27 @@ import GenericDetails from "./GenericDetails";
 import GSes from "../assets/coc-gses.webp";
 import Members from "../assets/coc-members.webp";
 import Banner from "./Banner";
-
 import "./AboutUs.css";
+import Image from "./Image";
+import Details from "./details";
 
 export default function AboutUs() {
+  const details = Details;
+  const arr = [];
+
+  for (let i = 0; i < 30; i += 2) {
```

Comment:
This increment will not work if you have an odd number of members.

1. Please don't hard code the values. If we have the data, we know its length. Please use Array.length instead.
2. Please update this logic to accommodate the last person who is left out.